### PR TITLE
Euclideon Logo replaces the embedded watermark.

### PR DESCRIPTION
**Pull request by asunawu**
Resolved [AB#1064](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/1064).

Euclideon Logo replaces the embedded watermark.
Added background in the watermark fragment shader.

### CODE REVIEW

**Comment by asunawu** [comment]
Maybe it is better to create a transparent background texture and rendering it under the logo.

File: src/vcRender.cpp [hunk 1]
```diff
@@ -1185,17 +1185,17 @@ bool vcRender_CreateSelectionBuffer(vcState *pProgramState, vcRenderContext *pRe
   return true;
 }
 
-void vcRenderWatermark(vcState *pProgramState, vcRenderContext *pRenderContext)
+void vcRenderWatermark(vcState *pProgramState, vcRenderContext *pRenderContext, vcTexture *pWatermark)
```
[the remark this answers is not in the document]
Comment (asunawu) [reply src/vcRender.cpp]: Fixed.

**Comment by asunawu** [comment]
@OsiJr Did you mean 'vcGLState_SetDepthStencilMode(vcGLSDM_Always, false);'? It has been added.
If I use imageRendererFragmentShader, it looks like this:

**Comment by peterAdams1014** [comment]
I've grabbed this branch and made some modifications. Turns out a shader source change was unnecessary, however I did make some vertex layout/buffer changes.

Also fixed an issue where the watermark was not appearing in screenshots.

**Pull request merged**